scripts/migration/migrate_date_modified_for_existing_nodes.py

```diff
@@ -25,7 +26,7 @@ def main():
 
     logger.warn('Date_modified field will be added to all nodes.')
 
-    for node in models.Node.find():
+    for node in models.Node.find(Q('date_modified', 'eq', None)):
```

Comment:
It seems to me like `Node.find(Q('date_modified', 'eq', None))` returns nodes that don't have date_modified at all.